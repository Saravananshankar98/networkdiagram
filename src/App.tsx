import "./App.css";
import HierarchicalModel from "./pages/herarical-view";
import ConnectorsPage from "./pages/connectors";
import MindMap from "./pages/mind-map";
import Radial from "./pages/radial-tree";
import NetworkShapes from "./pages/diagram";
import SimpleDiagram from "./pages/simple-diagram";
import Tooltip from "./pages/sample-diagram";

function App() {
  return (
    <div className="App">
      {/* <h1>Network Diagram</h1> */}
      {/* <NetworkShapes/> */}
      {/* <Tooltip/> */}
      <SimpleDiagram />
{/* <HierarchicalModel/> */}
      {/* <MindMap/> */}
      {/* <Radial/> */}
    </div>
  );
}

export default App;
