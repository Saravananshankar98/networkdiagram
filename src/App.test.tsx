import { render } from "@testing-library/react";
import App from "./App";

describe("App", () => {
  it("match snapshot", async () => {
    const { baseElement } = render(<App />);
    expect(baseElement).toMatchSnapshot();
  });
});
