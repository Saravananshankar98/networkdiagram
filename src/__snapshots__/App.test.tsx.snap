// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App match snapshot 1`] = `
<body>
  <div>
    <div
      class="App"
    >
      <div
        class="App"
      >
        <div>
          <div
            class="e-control e-diagram e-lib e-droppable e-tooltip"
            id="diagram"
            style="width: 100%; height: 800px; position: relative; overflow: hidden;"
            tabindex="0"
          >
            <div
              id="diagramcontent"
              style="width: 100%; height: 800px; position: absolute; left: 0px; top: 0px; overflow: auto; background: transparent;"
              tabindex="0"
            >
              <svg
                height="0"
                id="diagram_backgroundLayer_svg"
                style="position: absolute; top: 0px; left: 0px; overflow: hidden; pointer-events: none;"
                width="0"
              >
                <g
                  class="e-background-image-layer"
                  id="diagram_backgroundImageLayer"
                />
                <g
                  class="e-background-layer"
                  id="diagram_backgroundLayer"
                >
                  <rect
                    fill="transparent"
                    height="0"
                    id="diagram_backgroundLayerrect"
                    opacity="1"
                    pointer-events="none"
                    rx="0"
                    ry="0"
                    stroke-dasharray="0"
                    stroke-width="1"
                    transform="rotate(0,0,0)"
                    visibility="visible"
                    width="0"
                    x="0"
                    y="0"
                  />
                </g>
              </svg>
              <svg
                class="e-grid-layer"
                height="0"
                id="diagram_gridline_svg"
                style="position: absolute; top: 0px; left: 0px; overflow: hidden; pointer-events: none;"
                width="0"
              >
                <g
                  id="diagram_gridline"
                  transform="translate(0,0)"
                >
                  <rect
                    fill="url(#diagram_pattern)"
                    height="100%"
                    id="diagram_grid_rect"
                    width="100%"
                    x="0"
                    y="0"
                  />
                </g>
                <defs
                  id="diagram_grid_pattern_defn"
                >
                  <pattern
                    height="100"
                    id="diagram_pattern"
                    patternUnits="userSpaceOnUse"
                    width="100"
                    x="0"
                    y="0"
                  />
                </defs>
              </svg>
              <div
                id="diagram_diagramLayer_div"
                style="width: 0px; height: 0px; position: absolute; top: 0px; left: 0px; overflow: hidden; pointer-events: none;"
              >
                <svg
                  class="e-diagram-layer"
                  height="0"
                  id="diagram_diagramLayer_svg"
                  style="pointer-events: all;"
                  width="0"
                >
                  <g>
                    <g
                      id="diagram_diagramLayer"
                      transform="translate(0,0),scale(1)"
                    />
                  </g>
                </svg>
                <svg
                  class="e-native-layer"
                  height="0"
                  id="diagram_nativeLayer_svg"
                  style="position: absolute; top: 0px; left: 0px; overflow: hidden; pointer-events: none;"
                  width="0"
                >
                  <g
                    id="diagram_nativeLayer"
                    style="pointer-events: all;"
                    transform="translate(0,0),scale(1)"
                  />
                </svg>
              </div>
              <div
                class="e-html-layer"
                id="diagram_htmlLayer"
                style="width: 0px; height: 0px; position: absolute; top: 0px; left: 0px; overflow: hidden; pointer-events: none;"
              >
                <div
                  id="diagram_htmlLayer_div"
                  style="position: absolute; top: 0px; left: 0px; pointer-events: all; transform: translate(0px,0px)scale(1);"
                />
              </div>
              <svg
                class="e-ports-expand-layer"
                height="0"
                id="diagram_diagramPorts_svg"
                style="position: absolute; top: 0px; left: 0px; overflow: hidden; pointer-events: none;"
                width="0"
              >
                <g
                  class="e-ports-layer"
                  id="diagram_diagramPorts"
                  style="pointer-events: all;"
                  transform="translate(0,0),scale(1)"
                />
                <g
                  class="e-expand-layer"
                  id="diagram_diagramExpander"
                  style="pointer-events: all;"
                  transform="translate(0,0),scale(1)"
                />
              </svg>
              <div
                id="diagram_diagramAdornerLayer"
                style="width: 0px; height: 0px; position: absolute; top: 0px; left: 0px; overflow: hidden; pointer-events: none;"
              >
                <div
                  class="e-userHandle-layer"
                  id="diagram_diagramUserHandleLayer"
                  style="width: 0px; height: 0px; position: absolute; top: 0px; left: 0px; overflow: hidden; pointer-events: none;"
                />
                <svg
                  class="e-adorner-layer"
                  height="0"
                  id="diagram_diagramAdorner_svg"
                  style="pointer-events: none;"
                  width="0"
                >
                  <g
                    id="diagram_diagramAdorner"
                  >
                    <g
                      id="diagram_SelectorElement"
                    />
                  </g>
                </svg>
              </div>
            </div>
            <div
              class="diagram_hiddenUserHandleTemplate"
              style="width: 0px; height: 0px; visibility: hidden; overflow: hidden;"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    id="measureElement"
    style="visibility: hidden; height: 0px; width: 0px; overflow: hidden;"
  >
    <span
      style="display: inline-block; line-height: normal;"
    />
    <img
      alt="measureElementImage"
      src="data:image/gif;base64,R0lGODlhAQABAIAAAAAAAP///yH5BAEAAAAALAAAAAABAAEAAAIBRAA7"
    />
    <svg
      xmlns:xlink="http://www.w3.org/1999/xlink"
    >
      <path
        d=""
      />
      <text
        xml:space="preserve"
      />
    </svg>
  </div>
</body>
`;
